addons/l10n_ir/__init__.py

```diff
@@ -0,0 +1,3 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
```

Comment:
I think that for the translation of the COA to be present in the PO file you need to do this in the init file https://github.com/odoo/odoo/blob/15.0/addons/l10n_be/__init__.py :smile: